src/vidore_benchmark/evaluation/interfaces.py

```diff
@@ -0,0 +1,180 @@
+from datetime import datetime
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+
+
+class MetricsModel(BaseModel):
```

Comment:
Problem with this is that  might break in fure mteb releases ?

Reply:
You're right, plus the class is a bit bloated tbh... I'll simply make `MetricsModel` a `Dict[str, Optional[float]]`!